registries/_extension/x-oai-traits.md

```diff
@@ -0,0 +1,22 @@
+---
+owner: ralfhandl
+issue:
+description: Identify in an OpenAPI Description where Overlay updates should be applied.
```

Comment:
I'm not sure this captures what we're trying to say here. I think we are mostly using the trait to give something for a target expression to aim for - it's not about what's applied